File: 02_Programming_Fundamentals/02_Programming_Fundamentals_with_C#/03_Intro_And_Basic_Syntax_More_Exercise/03_gaming_store.cs
using System;

class MainClass {
  public static void Main (string[] args) {

  double moneyBalance = double.Parse(Console.ReadLine());
  double moneySpend = 0;
  string input = "";

  while(true) {
    input = Console.ReadLine();
    if (input == "Game Time") { 
      break; 
    } else if (moneyBalance <= 0) {
      break;
    } else if (input == "OutFall 4") { 
      if (moneyBalance >= 39.99) {
        moneyBalance -= 39.99; moneySpend += 39.99;
        Console.WriteLine("Bought OutFall 4");
      } else {
        Console.WriteLine("Too Expensive");
      }
    } else if (input == "CS: OG") { 
      if (moneyBalance >= 15.99) {
        moneyBalance -= 15.99; moneySpend += 15.99;
        Console.WriteLine("Bought CS: OG");
      } else {
        Console.WriteLine("Too Expensive");
      }
    } else if (input == "Zplinter Zell") { 
      if (moneyBalance >= 19.99) {
        moneyBalance -= 19.99; moneySpend += 19.99;
        Console.WriteLine("Bought Zplinter Zell");
      } else {
        Console.WriteLine("Too Expensive");
      }
    } else if (input == "Honored 2") { 
      if (moneyBalance >= 59.99) {
        moneyBalance -= 59.99; moneySpend += 59.99;
        Console.WriteLine("Bought Honored 2");
      } else {
        Console.WriteLine("Too Expensive");
      }
    } else if (input == "RoverWatch") { 
      if (moneyBalance >= 29.99) {
        moneyBalance -= 29.99; moneySpend += 29.99;
        Console.WriteLine("Bought RoverWatch");
      } else {
        Console.WriteLine("Too Expensive");
      }
    } else if (input == "RoverWatch Origins Edition") { 
      if (moneyBalance >= 39.99) {
        moneyBalance -= 39.99; moneySpend += 39.99;
        Console.WriteLine("Bought RoverWatch Origins Edition");
      } else {
        Console.WriteLine("Too Expensive");
      }
    } else { 
      Console.WriteLine("Not Found"); 
    }
  
  }

  if (moneyBalance <= 0) { 
    Console.WriteLine("Out of money!"); 
  } else {
    Console.WriteLine($"Total spent: ${moneySpend}. Remaining: ${moneyBalance:F2}");
  }
  
  }
}

File: 02_Programming_Fundamentals/04_Programming_Fundamentals_PHP/03_Intro_And_Basic_Syntax_More_Exercise/03_gaming_store.php
<?php

$money = floatval(readline());
$moneyInitial = $money;
while (true) {
  $game = readline();
  if ($game == "Game Time") { break; }
  if ($money <= 0) { break; }

  if ($game == "OutFall 4") {
    if ($money >= 39.99) {
      $money -= 39.99;
      echo "Bought OutFall 4" . PHP_EOL;
    } else {
      echo "Too Expensive" . PHP_EOL;
      continue;
    }
  } else if ($game == "CS: OG") {
    if ($money >= 15.99) {
      $money -= 15.99;
      echo "Bought CS: OG" . PHP_EOL;
    } else {
      echo "Too Expensive" . PHP_EOL;
      continue;
    }
  } else if ($game == "Zplinter Zell") {
    if ($money >= 19.99) {
      $money -= 19.99;
      echo "Bought Zplinter Zell" . PHP_EOL;
    } else {
      echo "Too Expensive" . PHP_EOL;
      continue;
    }
  } else if ($game == "Honored 2") {
    if ($money >= 59.99) {
      $money -= 59.99;
      echo "Bought Honored 2" . PHP_EOL;
    } else {
      echo "Too Expensive" . PHP_EOL;
      continue;
    }
  } else if ($game == "RoverWatch") {
    if ($money >= 29.99) {
      $money -= 29.99;
      echo "Bought RoverWatch" . PHP_EOL;
    } else {
      echo "Too Expensive" . PHP_EOL;
      continue;
    }
  } else if ($game == "RoverWatch Origins Edition") {
    if ($money >= 39.99) {
      $money -= 39.99;
      echo "Bought RoverWatch Origins Edition" . PHP_EOL;
    } else {
      echo "Too Expensive" . PHP_EOL;
      continue;
    }
  } else {
    echo "Not Found" . PHP_EOL;
  }

}

if ($money <= 0) {
  echo "Out of money!" . PHP_EOL;
} else {
  $money = number_format($money, 2, ".", "");
  $moneyDiff = number_format($moneyInitial - $money, 2, ".", "");
  echo "Total spent: $$moneyDiff. Remaining: $$money";  
}

?>